internal/openshift/controller.go

```diff
@@ -104,6 +112,11 @@ func (c *controllerImpl) HandleBuild(o model.Object) error {
 		"name": user.Name,
 	})
 
+	if disable := c.isUserDisabled(user.Name); disable {
```

Comment:
why not  just `if c.isUserDisabled(user.Name)` ?

Reply:
Done